<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminlogin</class>
 <widget class="QDialog" name="adminlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>[+] status</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>Sign In</string>
   </property>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 53, 97);
font: 700 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="styleSheet">
        <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>  UserName  :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_username"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_password">
       <property name="styleSheet">
        <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Password   :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resourse.qrc">:/img/images/loginlogo.png</pixmap>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}
QProgressBar::chunk {
    background-color: #CD96CD;
    width: 10px;
    margin: 0.5px;
}
</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_value">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
